Gemfile.lock

```diff
@@ -403,11 +407,10 @@ GEM
     websocket-extensions (0.1.5)
     whenever (1.0.0)
       chronic (>= 0.6.3)
-    zeitwerk (2.6.1)
+    zeitwerk (2.6.2)
 
 PLATFORMS
-  ruby
```

Comment:
I'm not sure if we need this platform or not. Do you recall? If not, what would you think about deploying this branch to QA and running an integration test to confirm we don't need the `ruby` platform in deployed envs?

Reply:
Oh wait, I did and it worked.

Reply:
OK, great. 🎉